### PR TITLE
[DOC-10493] New regions for Dedicated on AWS

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10493] New regions for Dedicated on AWS

### CODE REVIEW

File: src/current/cockroachcloud/regions.md [hunk 1]
```diff
@@ -47,22 +47,28 @@ CockroachDB {{ site.data.products.dedicated }} clusters can be deployed in the f
 
 Geographic Area | Region Name      | Location
 ----------------|------------------|---------
-Asia Pacific    | `ap-northeast-1` | Tokyo
+Asia Pacific    | `ap-east1`       | Hong Kong
```
Comment (angeladietz) [inline src/current/cockroachcloud/regions.md:50]:
```suggestion
Asia Pacific    | `ap-east-1`       | Hong Kong
```

**Review by angeladietz — APPROVED** [review]
Just a missing "-" but other than that it looks good, thank you!

**Review by mikeCRL — APPROVED** [review]

File: src/current/_includes/releases/cloud/2024-06-12.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# June 12, 2024
+
+<h3 id="2024-06-12-general-updates"> General updates </h3>
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/_includes/releases/cloud/2024-06-12.md:3]: They are _all_ this way, if you take a look, and there is no effect on the output. I've just left it alone, assuming there was something along the way stripping the space. Rather taking this edit here, maybe we should consider whether we want to make the change to all of them at once, then when we copy an existing file to create a new one, we won't need to do it again.

File: src/current/cockroachcloud/regions.md [hunk 3]
```diff
@@ -47,22 +47,28 @@ CockroachDB {{ site.data.products.dedicated }} clusters can be deployed in the f
...
                 | `ap-southeast-1` | Singapore
                 | `ap-southeast-2` | Sydney
-Middle East     | `me-central-1`   | United Arab Emirates
-North America   | `ca-central-1`   | Central Canada
+Middle East     | `me-south-1`     | Bahrain
+                | `il-central-1`   | Tel Aviv
+                | `me-central-1`   | United Arab Emirates
+North America   | `ca-west-1`      | Calgary
+                | `ca-central-1`   | Central Canada
                 | `us-east-1`      | N. Virginia
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/cockroachcloud/regions.md:63]: Let's not change it here. I can't figure out if either of those pages is nore authoritative than the other or if either one if them is more authoritative than https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/. Between the three, two of them use "N. Virginia". WDYT?

[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/cockroachcloud/regions.md:63]: Angela confirms that the AWS SDK (which influences the region names in CC) returns "N. Virginia".

**Pull request merged**